<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>436</width>
    <height>529</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>LitsModer</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="fileSelector_groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>411</width>
      <height>101</height>
     </rect>
    </property>
    <property name="title">
     <string>FileSelector</string>
    </property>
    <widget class="QToolButton" name="fileSelector_fileButton_toolButton">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>60</y>
       <width>26</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>...</string>
     </property>
    </widget>
    <widget class="QComboBox" name="fileSelector_modpackClass_comboBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>261</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="fileSelector_filePath_lineShow">
     <property name="geometry">
      <rect>
       <x>12</x>
       <y>60</y>
       <width>351</width>
       <height>21</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
     <property name="placeholderText">
      <string> File path</string>
     </property>
    </widget>
    <widget class="QPushButton" name="fileSelector_clearFilePath_pushButton">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>30</y>
       <width>131</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>Clear</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="scanner_groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>411</width>
      <height>111</height>
     </rect>
    </property>
    <property name="title">
     <string>Scanner</string>
    </property>
    <widget class="QPushButton" name="scanner_scan_pushButton">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>30</y>
       <width>131</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>Scan</string>
     </property>
    </widget>
    <widget class="QPushButton" name="scanner_resultShow_pushButton">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>30</y>
       <width>131</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>Result</string>
     </property>
    </widget>
    <widget class="QProgressBar" name="scanner_completeValue_progressBar">
     <property name="geometry">
      <rect>
       <x>17</x>
       <y>80</y>
       <width>371</width>
       <height>23</height>
      </rect>
     </property>
     <property name="value">
      <number>0</number>
     </property>
    </widget>
    <widget class="QLineEdit" name="scanner_processingTask_lineShow">
     <property name="geometry">
      <rect>
       <x>22</x>
       <y>60</y>
       <width>361</width>
       <height>21</height>
      </rect>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
     <property name="placeholderText">
      <string>Processing task</string>
     </property>
    </widget>
    <widget class="QPushButton" name="scanner_reset_pushButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>131</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>Reset</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>436</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
